omnibus/config/patches/chef-dk/cli.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/lib/chef-dk/cli.rb b/lib/chef-dk/cli.rb
+index d789b075..adfc498e 100644
+--- a/lib/chef-dk/cli.rb
++++ b/lib/chef-dk/cli.rb
+@@ -95,9 +95,10 @@ BANNER
+     end
+ 
+     def show_version
+-      msg("Chef Development Kit Version: #{ChefDK::VERSION}")
++      msg("Chef Workstation Version: CHEF_WS_VERSION")
```

Comment:
The former subversion user in me thinks that adding dollar signs here—`$CHEF_WS_VERSION$`—would make it more clear when looking at only this patch file that this is going to be replaced.